packages/examples/scripts/aws-test/coingecko-aws-test.ts

```diff
@@ -0,0 +1,68 @@
+import { readFileSync, writeFileSync } from 'fs';
+import { join } from 'path';
+import dotenv from 'dotenv';
+import { formatSecrets, cliPrint, runAndHandleErrors, runShellCommand } from '../../src';
+
+interface Secrets {
+  // Make sure the mnemonic is funded
+  MNEMONIC: string;
+  RINKEBY_PROVIDER_URL: string;
+  AWS_ACCESS_KEY_ID: string;
+  AWS_SECRET_ACCESS_KEY: string;
+  AWS_SESSION_TOKEN?: string;
+}
+
+let secrets: Secrets;
+
+const chooseIntegration = () => {
+  // We can't use the interactive script to choose the integration, so we specify the details manually
+  const content = JSON.stringify(
+    {
+      integration: 'coingecko',
+      airnodeType: 'aws',
+      network: 'rinkeby',
+      mnemonic: secrets.MNEMONIC,
+      providerUrl: secrets.RINKEBY_PROVIDER_URL,
+    },
+    null,
+    2
+  );
+  writeFileSync(join(__dirname, '../../integration-info.json'), content);
```

Comment:
Is there a cleanup step at the end of running the tests to remove this file since it seems it might contain sensitive information?

Reply:
Yes, I guess that should be enough 💯 
I'm now just wondering if these files are really needed 🤔 Wouldn't it be better to have a helper ts file to read these values from evn vars in all other scripts instead? Maybe I'm missing something that forces us to have them in json files

Reply:
Thank you so much for your detailed explanation.